src/components/Registration/DefineRegistrationSummary.vue

```diff
@@ -155,6 +171,11 @@ export default class DefineRegistrationSummary extends Mixins(DateMixin, EnumMix
     return this.getRegistration.businessNumber
   }
 
+  /** The business type confirm. */
+  get businessTypeConfirm (): boolean {
+    return this.getRegistration.businessTypeConfirm
+  }
```

Comment:
I added this method in my latest commit, for the same reason above. I am not sure I should add this.

Reply:
Looks fine to me.